NEWS.md

```diff
@@ -1,12 +1,12 @@
-# wwinference 0.0.1
+# wwinference 0.0.1 (dev)
```

Comment:
This `(dev)` tag is just extra information that can be removed once the first version is released.